.classpath

```diff
@@ -7,5 +7,6 @@
 		</attributes>
 	</classpathentry>
 	<classpathentry kind="con" path="org.eclipse.jdt.junit.JUNIT_CONTAINER/5"/>
+	<classpathentry kind="lib" path="/home/sravan/Downloads/json-20200518.jar"/>
```

Comment:
@muhammedyaseen-git @sravand123 We need to have this jar in one of the directories.